group/ristretto255.go

```diff
@@ -0,0 +1,179 @@
+package group
+
+import (
+	"crypto"
+	"io"
+
+	h2c "github.com/armfazh/h2c-go-ref"
+
+	r225 "github.com/bwesterb/go-ristretto"
+)
+
+var (
+	Ristretto255 Group = ristrettoGroup{}
+)
+
+type ristrettoGroup struct {
+}
+
+type ristrettoElement struct {
+	p *r225.Point
+}
```

Comment:
I'm surprised by the choice to have a pointer inside a struct.  Why not

```go
type ristrettoElement struct {
 	p r225.Point
 }
```

or the closely related

```go
type ristrettoElement r255.Point
```

Reply:
No particular reason. I changed this to make it the former.